src/installing-deis/installing-deis-workflow.md

```diff
@@ -46,7 +46,8 @@ $ helm fetch deis/deis
 Now that you have Helm installed and added the Deis Chart Repository, install Workflow by running:
 
 ```
-$ helm install deis/deis --namespace=deis
+$ helm generate -x manifests deis      # generates various secrets
+$ helm install deis --namespace=deis   # injects resources into your cluster
```

Comment:
We don't need to specify `--namespace=deis` here since the chart includes it. (It's only `helm uninstall` that needs hand-holding.)